developer-portal-wrapper/developer_portal_wrapper.py

```diff
@@ -142,13 +143,19 @@ def parse_args():
 
 def main():
     args = parse_args()
-    validate_env_vars()
+
+    loglevel = logging.DEBUG if args.debug else logging.INFO
+
+    stream_handler = logging.StreamHandler(sys.stdout)
+    stream_handler.setLevel(loglevel)
 
     logging.basicConfig(
         level=logging.DEBUG if args.debug else logging.INFO,
```

Comment:
You missed this one from my PR ;)
```suggestion
        level=loglevel,
```

Reply:
fixed. thanks